### DISCUSSION TITLE
Providers without NPI

### DISCUSSION THREAD

**Discussion by Rashmi-sb** [CMSgov/price-transparency-guide#530] [Q&A] [answered]
We are faced with a scenarios were we do not capture NPI for providers .Should these providers be part of In Network file  ?
If yes, how do we indicate these providers in file any alternate values to be used ?

**Comment by RLTx1391** [comment, marked as answer]
For these types of contracts, please use a single "0" in the NPI array 
https://github.com/CMSgov/price-transparency-guide/discussions/457

**Reply by Rashmi-sb** [reply]
In our scenarios all the providers are saved without NPI, with this suggestion , we would be displaying "0" for all the providers . Is this accepted by CMS ?

**Reply by sbenphia** [reply]
If there is a 0 in NPI, does that mean there would be an EIN?  Or are there going to be databases out there with blanks in both?

**Reply by tdfow** [reply]
@sbenphia, the expectation is the Provider will have a tax id.  The only condition that would have the '0' NPI and no tax id is if the Provider is using their SSN versus an EIN.  If the Provider has an SSN, then the tax id type is 'npi'--so if they don't have an NPI, then both tax id value of 'npi' type and the 'npi' would both be 0---and not have any meaning/relevance.
If the Provider has EIN and no NPI, then the EIN is published with the NPI equal to 0
